### PR TITLE
Feature/fm 679 da sending the contract

### PR DESCRIPTION
**Pull request by ramchurnk**

### CODE REVIEW

**Comment by ramchurnk** [comment]
Deleted schema due to questions arising over changes which had not been made on my part - re-saved/added schema back in due to being told that deleting it was a mistake and the edits were to have come from someone else

**Comment by iceraluk** [comment]
just remove the changes added into the schema and re-commit it. it should be the same as master

File: config/routes.rb [hunk 1]
```diff
@@ -167,6 +167,10 @@
         get 'contract-summary/declined-offer', to: 'contract_summary#declined_offer'
         get 'contract-summary/not-responded-to-contract-offer', to: 'contract_summary#not_responded_to_contract_offer'
       end
+      namespace :direct_award_contract, path: 'direct-award-contract' do
```
Comment (iceraluk) [inline config/routes.rb]: not sure if you need a new namespace for this? the following should be enough:
get 'sending-the-contract', to: 'direct_award_contract#sending_the_contract'

File: db/schema.rb [hunk 2]
```diff
@@ -454,16 +452,6 @@
     t.index ["postcode"], name: "idx_postcode"
   end
 
-  create_table "os_address_admin_uploads", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```
Comment (iceraluk) [inline db/schema.rb]: not sure where this is coming from. but we shouldn't have schema changes without migrations. just match the schema with master and re-commit.

Comment (ramchurnk) [reply db/schema.rb]: done

File: config/routes.rb (hunk 1 above)
Comment (ramchurnk) [reply config/routes.rb]: Done, as discussed

File: app/views/facilities_management/beta/direct_award_contract/_link_to_start_page.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<span class="govuk-heading-m fm-buyer-account-heading"> Find facilities management services </span>
```
Comment (iceraluk) [inline app/views/facilities_management/beta/direct_award_contract/_link_to_start_page.html.erb]: this is probably not needed anymore

Comment (ramchurnk) [reply app/views/facilities_management/beta/direct_award_contract/_link_to_start_page.html.erb]: yup, deleted.

**Pull request merged**